### ISSUE TITLE
UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined - rollup v1.0.0

### ISSUE THREAD

**Issue by btakita** [sveltejs/sapper#540]
Looks like there's an error upon upgrading rollup to 1.0.0.

`UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined`

which happens in

`node_modules/sapper/dist/core.js:573:40`

`_a.sent()` now returns `{ output: { code: '...' } }` instead of the expected `{ code: '...' }`

**Comment by laurentpayot** [comment]
+1, same error here:
```
(node:4212) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at stripShebang (internal/modules/cjs/helpers.js:69:25)
    at Module._compile (internal/modules/cjs/loader.js:679:13)
    at Object.require.extensions..js (/home/laurent.payot/projects/myproject-sapper/node_modules/sapper/dist/core.js:573:40)
    at Module.load (internal/modules/cjs/loader.js:620:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:560:12)
    at Function.Module._load (internal/modules/cjs/loader.js:552:3)
    at Module.require (internal/modules/cjs/loader.js:657:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Function.<anonymous> (/home/laurent.payot/projects/myproject-sapper/node_modules/sapper/dist/core.js:579:34)
    at step (/home/laurent.payot/projects/myproject-sapper/node_modules/sapper/node_modules/tslib/tslib.js:133:27)
(node:4212) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:4212) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by Conduitry** [comment]
Fixed in Sapper 0.24.1 and in the latest `sapper-template#rollup`.
